### PR TITLE
fix(dockerdev): Configure caching for Docker volumes

### PR DESCRIPTION
**Pull request by oliverklee**
This should improve performance, particularly on macOS.

### CODE REVIEW

File: examples/dockerdev/docker-compose.yml [hunk 1]
```diff
@@ -71,8 +71,8 @@ services:
   postgres:
     image: postgres:13.0
     volumes:
-      - .dockerdev/.psqlrc:/root/.psqlrc:ro
-      - postgres:/var/lib/postgresql/data
+      - .dockerdev/.psqlrc:/root/.psqlrc:ro,cached
+      - postgres:/var/lib/postgresql/data:delegated
       - ./log:/root/log:cached
```
Comment (jandudulski) [inline examples/dockerdev/docker-compose.yml]: Shouldn't be delegated?

File: examples/dockerdev/docker-compose.yml [hunk 2]
```diff
@@ -27,13 +27,13 @@ x-backend: &backend
...
-    - packs-test:/app/public/packs-test
-    - .dockerdev/.psqlrc:/root/.psqlrc:ro
-    - .dockerdev/.bashrc:/root/.bashrc:ro
+    - rails_cache:/app/tmp/cache:delegated
+    - bundle:/usr/local/bundle:delegated
+    - node_modules:/app/node_modules:delegated
+    - packs:/app/public/packs:delegated
+    - packs-test:/app/public/packs-test:delegated
+    - .dockerdev/.psqlrc:/root/.psqlrc:ro,cached
+    - .dockerdev/.bashrc:/root/.bashrc:ro,cached
```
Comment (jandudulski) [inline examples/dockerdev/docker-compose.yml:36]: I wonder if `log` shouldn't be listed here too?

Comment (oliverklee) [reply examples/dockerdev/docker-compose.yml:36]: Yes, makes sense. Added. Thanks!

File: examples/dockerdev/docker-compose.yml (hunk 1 above)
Comment (oliverklee) [reply examples/dockerdev/docker-compose.yml]: Indeed. Updated.

**Comment by oliverklee** [comment]
Updated according to the review comments and repushed.

File: examples/dockerdev/docker-compose.yml [hunk 3]
```diff
@@ -27,13 +27,14 @@ x-backend: &backend
...
-    - node_modules:/app/node_modules
-    - packs:/app/public/packs
-    - packs-test:/app/public/packs-test
-    - .dockerdev/.psqlrc:/root/.psqlrc:ro
-    - .dockerdev/.bashrc:/root/.bashrc:ro
+    - rails_cache:/app/tmp/cache:delegated
+    - bundle:/usr/local/bundle:delegated
+    - node_modules:/app/node_modules:delegated
+    - packs:/app/public/packs:delegated
+    - packs-test:/app/public/packs-test:delegated
```
Comment (palkan) [inline examples/dockerdev/docker-compose.yml:34]: Doesn't `delegated`,`cached` affect only volumes mounted from the host system? All of these "live" within a Docker.

**Comment by oliverklee** [comment]
Okay, according to https://github.com/docker/for-mac/issues/5402, this configuration does not seem to be necessary for good mount performance on macOS. Thanks!

Closing.

**Pull request closed without merging**